ingestion_server/test/integration_test.py

```diff
@@ -431,18 +432,9 @@ def tearDownClass(cls) -> None:
         # Terminate the Bottle process started in ``setUp``
         cls.cb_process.terminate()
 
-        # Stop all running containers and delete all data in volumes
-        compose_path = cls.compose_path
-        log_output = compose_path.parent / "ingestion_logs.txt"
-        with log_output.open("w") as file:
-            cls._compose_cmd(
-                ["logs", "--no-color"], stderr=subprocess.STDOUT, stdout=file
-            )
-
-        cls._compose_cmd(
-            ["down", "-v"],
-            capture_output=True,
-        )
+        # Tear down the Docker Compose stack locally
+        if "GITHUB_ACTIONS" not in os.environ:
```

Comment:
Just a nit: we usually check for the `CI` variable in these cases. This isn't a blocker, but using a consistent variable makes it more intuitive for us to test the differences between a CI run and local run by setting `CI=true` before running the command (vs needing to dig up what specific variable(s) the particular thing we're running checks).

```suggestion
        if "CI" not in os.environ:
```

Here are examples of our usage of the CI variable:

https://github.com//WordPress/openverse/blob/bdb8f0ad1dd0b2ae504f708b4105fdadc10e8988/justfile#L10

https://github.com//WordPress/openverse/blob/fae1dc0ef5f49c6f424db5c018e328e8d8e91f15/frontend/nuxt.config.ts#L71

https://github.com//WordPress/openverse/blob/9471728b4686d054493cd4528554e957dfee4350/frontend/test/playwright/playwright.config.ts#L29

I think CI is a bit more standardised, if we switched CI runners away from GitHub actions (what a mess that would be :sweat_smile:) those checks would continue working without modification.

Reply:
Thank you for this. Made the update :)